<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formProverkaAll</class>
 <widget class="QWidget" name="formProverkaAll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_tableOfTrue_result">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>7</y>
     <width>236</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>текст</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_resultAll_result">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>239</y>
     <width>236</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>текст</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_skleyka2_result">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>94</y>
     <width>236</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>текст</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_kartaPokritiya_result">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>152</y>
     <width>236</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>текст</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_skleyka3_result">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>123</y>
     <width>236</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>текст</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_resultAll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>239</y>
     <width>239</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Результат: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_skleyka1_result">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>65</y>
     <width>236</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>текст</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_levelOfHard">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>239</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Уровень сложности: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tableOfTrue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>7</y>
     <width>239</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Таблица истинности: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_minNf_result">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>181</y>
     <width>236</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>текст</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_viborZnacheniy">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>36</y>
     <width>239</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Выбор значений: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_skleyka3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>123</y>
     <width>239</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Склейка 3:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_kartaPokritiya">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>152</y>
     <width>239</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Карта покрытия:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_skleyka1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>239</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Склейка 1:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_viborZnacheniy_result">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>36</y>
     <width>236</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>текст</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_minNf">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>181</y>
     <width>239</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Минимальная функция:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_skleyka2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>94</y>
     <width>239</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Склейка 2:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_levelOfHard_result">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>236</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>текст</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
